jekyll/_cci2/configuration-reference.md

```diff
@@ -539,6 +539,48 @@ The directory layout should match the [classic CircleCI test metadata directory
     path: /tmp/test-results
 ```
 
+#### **`persist_to_workspace`**
+
+Special step used to persist a temporary file to be used by another job in the workflow. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+paths | Y | String | Directory containing temporary files
+{: class="table table-striped"}
+
+##### _Example_
+
+``` YAML
+- persist_to_workspace:
+    paths: /tmp/file-name
+```
+
+#### **`attach__workspace`**
+
+Special step used to attach a temporary file generated by another job in the workflow. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+at | Y | String | Directory containing temporary files
+{: class="table table-striped"}
+
+##### _Example_
+
+``` YAML
+- attach_workspace:
+    at: /tmp/file-name
+```
+
+Note the following distinctions between Artifacts, Workspaces, and Caches:
+
+| Type      | lifetime        | Use                      | Example |
```

Comment:
I love the idea of this comparison table.  Just last night I had a conversation with a user about why caches and workspaces both exist.